### ISSUE TITLE
Improving semantic html of default templates

### ISSUE THREAD

**Issue by vnglst** [phoenixframework/phoenix#4347]
Hi everyone!

I just started learning Phoenix using the `Programming Phoenix` book and I love it so far!

I'm a frontend developer by day and I noticed something in the html of the standard templates that could be improved. It correctly uses semantic html elements like `main` and `nav`, but it also specifies the implicit aria roles of `main` and `nav`. 

I don't think that is necessary anymore (according to the [spec](https://html.spec.whatwg.org/multipage/grouping-content.html#the-main-element) the `main` element has an implicit aria role and you should not set any role). Explicitly setting the role used to be a workaround for older browsers (released before 2013).

Do you also see this as an improvement and is this something I can help fix? I'd love to help!

Basically I would suggest making the following change for `main` and `nav`:

```diff
-    <main role="main" class="container">
+   <main class="container">
      <p class="alert alert-info" role="alert"><%%= get_flash(@conn, :info) %></p>
      <p class="alert alert-danger" role="alert"><%%= get_flash(@conn, :error) %></p>
      <%%= @inner_content %>
    </main>
```

**Comment by josevalim** [comment]
Yes, please, a PR is welcome!

**Comment by vnglst** [comment]
That’s great news! I’ll open a PR as soon as I get back from my holidays.
